Jenkinsfile.SensorboxTests

```diff
@@ -1,17 +1,50 @@
 #!groovy
 pipeline {
-        agent {
-                node {
-                    label 'HardwareSensorBox'
-              }
-         }
+     agent {
+        dockerfile {
+            filename 'Dockerfile.jenkins'
+            // 'docker build' would normally copy the whole build-dir to the container, changing the
+            // docker build directory avoids that overhead
+            dir 'docker'
+            // run on slave1 where nexus is connected to
+            label 'DeviceHardware'
+            // Pass the uid and the gid of the current user (jenkins-user) to the Dockerfile, so a
+            // corresponding user can be added. This is needed to provide the jenkins user inside
+            // the container for the ssh-agent to work.
+            // Another way would be to simply map the passwd file, but would spoil additional information
+            additionalBuildArgs '--build-arg USER_ID=$(id -u) --build-arg GROUP_ID=$(id -g)'
+            args "--net host -v /dev/bus/usb/:/dev/bus/usb --device /dev/kvm:/dev/kvm -v /var/local/container_shared/gradle/:/.gradle -v /var/local/container_shared/android-sdk:/usr/local/android-sdk -v /var/local/container_shared/android-home:/.android -v /var/local/container_shared/emulator_console_auth_token:/.emulator_console_auth_token -v /var/local/container_shared/analytics.settings:/analytics.settings"
+        }
+    }
 
     environment {
+        //////// Define environment variables to point to the correct locations inside the container ////////
         //////////// Most likely not edited by the developer
-        ANDROID_SDK_ROOT = "${ANDROID_SDK_LOCATION}"
+        ANDROID_SDK_ROOT = "/usr/local/android-sdk"
         // Deprecated: Still used by the used gradle version, once gradle respects ANDROID_SDK_ROOT, this can be removed
-        ANDROID_HOME = "${ANDROID_SDK_LOCATION}"
-        ANDROID_SDK_LOCATION = "${ANDROID_SDK_LOCATION}"
+        ANDROID_HOME = "/usr/local/android-sdk"
+        ANDROID_SDK_HOME = "/"
+        // Needed for compatibiliby to current Jenkins-wide Envs. Can be removed, once all builds are migrated to Pipeline
+        ANDROID_SDK_LOCATION = "/usr/local/android-sdk"
+        ANDROID_NDK = ""
+        // This is important, as we want the keep our gradle cache, but we can't share it between containers
+        // the cache could only be shared if the gradle instances could comunicate with each other
+        // imho keeping the cache per executor will have the least space impact
+        GRADLE_USER_HOME = "/.gradle/${env.EXECUTOR_NUMBER}"
+        // Otherwise user.home returns ? for java applications
+        JAVA_TOOL_OPTIONS = "-Duser.home=/tmp/"
+
+        //// jenkins-android-helper related variables
+        // set to any value to debug jenkins_android* scripts
```

Comment:
This block, including ANDROID_EMULATOR_HELPER_DEBUG and PYTHONUNBUFFERED are not needed anymore.

Reply:
Seems fine to me with the changes. 
See https://jenkins.catrob.at/job/lab/job/Sandro_Eitzinger/job/SensorBox-in-Docker/67/